homepage.html

```diff
@@ -0,0 +1,118 @@
+                        <li class="list-items"><a href="#">Sponsor</a></li>
+                        <li class="list-items"><a href="#">News</a></li>
+                        <li class="list-items1 campaign"><a href="#">Cc Campaign</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section class="home-main">
+            <div class="main-cont">
+                <div class="hamburger-btn">
+                    <div class="ham-line"></div>
+                    <div class="ham-line"></div>
+                    <div class="ham-line"></div>
+                </div>
+                <p class="main-cont-p1">"What seest thou..."</p>
+                <h1>THE PLATFORM<br>NIGERIA<br>2021</h1>
+                <p class="main-cont-p2">The Platform Nigeria events are designed to facilitate growth in the areas of personal capacity, productivity & National Development within Nigeria. Nigeria.</p>
+                <h4>2021.10.01(FRI) ~ 02(SAT)</h4>
+                <p class="main-cont-p3">@ The Covenant Place, Iganmu (beside The National Theatre)</p>
+            </div>
+        </section>
+        <section class="main-program">
+            <h3>Main Program</h3>
+            <hr>
+            <div class="main-flex">
+                <article class="main-flex-item">
+                    <img src="images/lecture.png" alt="icon">
+                    <h5>Lecture</h5>
+                    <p>listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </article>
+                <article class="main-flex-item">
+                    <img src="images/exhibition.png" alt="icon">
+                    <h5>CC Exhibition</h5>
+                    <p>listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </article>
+                <article class="main-flex-item">
+                    <img src="images/forum.png" alt="icon">
+                    <h5>Forum</h5>
+                    <p>listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </article>
+                <article class="main-flex-item">
+                    <img src="images/workshop.png" alt="icon">
+                    <h5>Workshop</h5>
+                    <p>listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </article>
+                <article class="main-flex-item">
+                    <img src="images/ignite.png" alt="icon">
+                    <h5>CC Ignite</h5>
+                    <p>listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </article>
+            </div>
+            <u><a>SEE THE WHOLE PROGRAM</a></u>
+        </section>
+        <section class="feature-speaker">
+            <h1>Feature Speakers</h1>
```

Comment:
- [ ] There's at least 1, but no more than 1, `h1` tag on each page. (Please change this `h1` tag to something else to comply with the project requirements)